ignis/services/systemd/service.py

```diff
@@ -0,0 +1,41 @@
+from ignis.dbus import DBusProxy
+from ignis.utils import Utils
+from ignis.base_service import BaseService
+
+class SystemdService(BaseService):
+    """
+    A service for controlling systemd units through DBus.
+
+    Example usage:
+
+    .. code-block:: python
+
+        from ignis.services.systemd import SystemdService
+
+        systemd = SystemdService.get_default()
+
+        systemd.start_unit("rot8.service")
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.__dbus = DBusProxy(
+            name = "org.freedesktop.systemd1",
+            object_path = "/org/freedesktop/systemd1",
+            interface_name = "org.freedesktop.systemd1.Manager",
+            info = Utils.load_interface_xml("org.freedesktop.DBus"),
```

Comment:
You have to add a XML file that defines ``org.freedesktop.systemd1.Manager`` interface to ``ignis/dbus`` and load it instead of ``org.freedesktop.DBus``

Reply:
Thank you, that explains things. Sorry for the oversight..